subprojects/logging/src/main/java/org/gradle/internal/logging/console/ProgressOperation.java

```diff
@@ -16,7 +16,7 @@
 
 package org.gradle.internal.logging.console;
 
-import org.gradle.internal.logging.events.OperationIdentifier;
+import org.gradle.internal.progress.OperationIdentifier;
```

Comment:
Moving `OperationIdentifier` around isn't really necessary for this work. We should do that as a separate move.